src/main/java/pl/edu/mimuw/chess/ChessBoard.java

```diff
@@ -0,0 +1,129 @@
+package pl.edu.mimuw.chess;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.Random;
+
+import pl.edu.mimuw.chess.Util;
+
+public class ChessBoard {
+	public static final int BOARD_SIZE = 8;
+	
+	public static final String EMPTY_BOARD = "╔═╤═╤═╤═╤═╤═╤═╤═╗\n" +
+	                                         "║█│ │█│ │█│ │█│ ║\n" +
+	                                         "╟─┼─┼─┼─┼─┼─┼─┼─╢\n" +
+	                                         "║ │█│ │█│ │█│ │█║\n" +
+	                                         "╟─┼─┼─┼─┼─┼─┼─┼─╢\n" +
+	                                         "║█│ │█│ │█│ │█│ ║\n" +
+	                                         "╟─┼─┼─┼─┼─┼─┼─┼─╢\n" +
+	                                         "║ │█│ │█│ │█│ │█║\n" +
+	                                         "╟─┼─┼─┼─┼─┼─┼─┼─╢\n" +
+	                                         "║█│ │█│ │█│ │█│ ║\n" +
+	                                         "╟─┼─┼─┼─┼─┼─┼─┼─╢\n" +
+	                                         "║ │█│ │█│ │█│ │█║\n" +
+	                                         "╟─┼─┼─┼─┼─┼─┼─┼─╢\n" +
+	                                         "║█│ │█│ │█│ │█│ ║\n" +
+	                                         "╟─┼─┼─┼─┼─┼─┼─┼─╢\n" +
+	                                         "║ │█│ │█│ │█│ │█║\n" +
+	                                         "╚═╧═╧═╧═╧═╧═╧═╧═╝\n";
+	
+	public static final int BOARD_ROW_LENGTH = BOARD_SIZE * 2 + 2;
+	
+	public static boolean isOnBoard(V2 position) {
+		return 0 <= position.x && position.x < BOARD_SIZE &&
+		       0 <= position.y && position.y < BOARD_SIZE;
+	}
+	
+	private static final Random RANDOM = new Random();
+	
+	public static void play() {
+		Player whitePlayer = new Player(ChessColor.WHITE);
```

Comment:
to że to nie znalazło się w `main`, nie oznacza ze na pałe w jednej metodzie powinno się znależć xd - tu jest do wydzielenia conajmniej kilka abstrakcji, które poprawiłyby czytelność tego kodu